src/containers/signup-confirmation/signup-confirmation.less

```diff
@@ -0,0 +1,32 @@
+.content-container {
+  display: block;
+  padding: 24px;
+  margin: auto;
+  font-family: Roboto;
```

Comment:
Wait, we aren't using Roboto. I guess I didn't make this clear in the ticket though, we should remove this line anyway.

Reply:
Oh yeah that is totally my bad, that was from before I realized that font and colors changed when we got the hifi design and I forgot to change it. Sorry about that!